### DISCUSSION TITLE
Slowing down development due to life events.

### DISCUSSION THREAD

**Discussion by bitbound** [immense/Remotely#352] [General]
Hey, all.

I thought this is worth sharing, as it will probably become more evident here shortly.

I've recently accepted a new job at another company. At the same time, we're also getting close to closing on a house.

Both my day job and personal life will soon be requiring more of my time and energy. As a result, development on Remotely will be slowing down quite a bit.  However, I'll still be slowly working toward a functioning remote control client for Mac. 

I also won't have as much time for responding to emails, so I encourage anyone with questions to please post in Reddit or GitHub Discussions. There are a lot of extremely knowledgeable users who like to help out.

Thanks, and take care!

- Jared

**Comment by swilkey** [comment]
Congratulations Jared!
Thanks for all the work that you've been able to put into remotely 
lately - I haven't been able to keep up with all the changes in my 
installation but I'm looking forward to doing a big upgrade soon to 
benefit from the latest improvements.
Regards,
Stephen

------ Original Message ------
From: "Jared Goodwin" ***@***.***>
To: "lucent-sea/Remotely" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: 11/05/2021 7:12:06 AM
Subject: [lucent-sea/Remotely] Slowing down development due to life 
events. (#352)


Hey, all.

I thought this is worth sharing, as it will probably become more evident 
here shortly.

I've recently accepted a new job at another company. At the same time, 
we're also getting close to closing on a house.

Both my day job and personal life will soon be requiring more of my time 
and energy. As a result, development on Remotely will be slowing down 
quite a bit. However, I'll still be slowly working toward a functioning 
remote control client for Mac.

I also won't have as much time for responding to emails, so I encourage 
anyone with questions to please post in Reddit or GitHub Discussions. 
There are a lot of extremely knowledgeable users who like to help out.

Thanks, and take care!

Jared
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub 
<https://github.com/lucent-sea/Remotely/discussions/352>, or unsubscribe

**Comment by dtruck260** [comment]
Best of luck to you
